user/saver.c

```diff
@@ -42,26 +34,36 @@ int main(const char *argv[], int argc)
 	int dg = 2;
 	int db = 3;
 
-	draw_window(wd);
-	draw_clear(0, 0, WIDTH, HEIGHT);
+	int dims[2];
+	syscall_object_size(KNO_STDWIN, (int *)&dims, 2);
```

Comment:
Unusual cast -- why not just pass "dims"?

Reply:
Not sure why I did that.